addons/mail/models/mail_thread.py

```diff
@@ -3970,6 +3970,24 @@ def _notify_by_web_push_prepare_payload(self, message, msg_vals=False):
         else:
             icon = '/web/static/img/odoo-icon-192x192.png'
 
+        if not body and not message.body and message.attachment_ids:
+            total_attachments = len(message.attachment_ids)
+            first_attachment = message.attachment_ids[0]
+            first_attachment_label = _('Voice message') if first_attachment.voice_ids else first_attachment.name
```

Comment:
`voice_ids` is supposed to be accessible only with sudo. I'm assuming we are necessarily in a sudo context when calling this method, or with the `voice_ids` of the attachments being already in cache?